website/docs/user-group-role/access-control/permissions.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "About permissions"
+description: "Learn about global and object permissions in authentik."
+---
+
+Permissions are the central components in all access control systems, the lowest-level component, the controlling piece of access data. Permissions are assigned to (or removed from!) to define exactly WHO can do WHAT to WHICH part of the overall software system.
```

Comment:
We should probably also clarify the difference that these permissions only apply to objects within authentik and *not* to who can access certain applications since that's done with policies